### PR TITLE
Create build.yml

### PR DESCRIPTION
**Pull request by IlyaMoskva**
Issue #7 : Added CI based on Gradle

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Build git-stat
+
+on:
+  pull_request:
+    branches: [ main ]
```
Comment (g4s8) [inline .github/workflows/build.yml:5]: @IlyaMoskva we have `master` primary branch

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Build git-stat
```
Comment (g4s8) [inline .github/workflows/build.yml:1]: @IlyaMoskva `git-stat` is not a correct project name

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Cache Gradle caches
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/
+            ~/.gradle/wrapper/
+          key: build-cache-gradle-${{ hashFiles('**/build.gradle*') }}
+          restore-keys: |
+            build-cache-gradle-
+      - name: Build git-stat
```
Comment (g4s8) [inline .github/workflows/build.yml:27]: @IlyaMoskva again: `git-stat`

**Review by g4s8 — CHANGES_REQUESTED** [review]
@IlyaMoskva please check my comments

File: .github/workflows/build.yml (hunk 1 above)
Comment (IlyaMoskva) [reply .github/workflows/build.yml]: Good catch, @g4s8 , Thanks. Fixed.

File: .github/workflows/build.yml (hunk 2 above)
Comment (IlyaMoskva) [reply .github/workflows/build.yml]: Fixed

File: .github/workflows/build.yml (hunk 3 above)
Comment (IlyaMoskva) [reply .github/workflows/build.yml]: Fixed

**Review by g4s8 — APPROVED** [review]
@IlyaMoskva thanks, let's wait for #11 PR to be merged to make run this test with gradle script

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
+name: Build degitx-simulator
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  diktat_check:
```
Comment (g4s8) [inline .github/workflows/build.yml:8]: @IlyaMoskva I just missed this name: let's fix this job name, now Github shows check named `diktat_check` in CI list

**Review by g4s8 — CHANGES_REQUESTED** [review]
@IlyaMoskva one missed line

File: .github/workflows/build.yml (hunk 4 above)
Comment (IlyaMoskva) [reply .github/workflows/build.yml]: Haha )) Thanks, @g4s8 , fixed as well.

**Review by g4s8 — APPROVED** [review]

**Pull request merged**